honeybee_standards/_util/_program_type.py

```diff
@@ -0,0 +1,177 @@
+                'Strip mall - type 1': 'Type 1',
+                'Strip mall - type 2': 'Type 2',
+                'Strip mall - type 3': 'Type 3',
+                'HospitalOfficeFlr1': 'HospitalOffice'}
+
+    # initialize the clean dictionaries
+    index_dict = {'LargeOffice': [], 'SmallOffice': [], 'MediumOffice': []}
+    program_type_dict = {}
+
+    # extract data from the raw standards gem json file
+    with open(source_filename, 'r') as f:
+        data_store = json.load(f)
+
+    for prog in data_store['space_types']:
+        # get the building type and subdivide office into small, medium, and large
+        bldg_type = prog['building_type']
+        if bldg_type not in index_dict and bldg_type not in _exclude_bldg:
+            index_dict[bldg_type] = []
+
+        if prog['space_type'] in _exclude:
+            # get rid of odd space types that are not generally aplicable
+            pass
+        elif prog['space_type'] in _replace:
+            # rename space types that use inconsistent naming conventions
+            index_dict[bldg_type].append(_replace[prog['space_type']])
+            new_name, clean_data = clean_individual_type(
+                vintage, bldg_type, _replace[prog['space_type']], prog)
+            program_type_dict[new_name] = clean_data
+        elif bldg_type == 'Office':
+            # reorganize office sace types by large, medium and small offices.
+            if prog['space_type'].startswith('WholeBuilding'):
+                pass
+            elif prog['space_type'].startswith('SmallOffice'):
+                prog_type = prog['space_type'].split(' - ')[-1]
+                index_dict['SmallOffice'].append(prog_type)
+                new_name, clean_data = clean_individual_type(
+                    vintage, 'SmallOffice', prog_type, prog)
+                program_type_dict[new_name] = clean_data
+            elif prog['space_type'].startswith('MediumOffice'):
+                prog_type = prog['space_type'].split(' - ')[-1]
+                index_dict['MediumOffice'].append(prog_type)
+                new_name, clean_data = clean_individual_type(
+                    vintage, 'MediumOffice', prog_type, prog)
+                program_type_dict[new_name] = clean_data
+            else:
+                index_dict['LargeOffice'].append(prog['space_type'])
+                new_name, clean_data = clean_individual_type(
+                    vintage, 'LargeOffice', prog['space_type'], prog)
+                program_type_dict[new_name] = clean_data
+        else:
+            # no need to change anything about the name
+            index_dict[bldg_type].append(prog['space_type'])
+            new_name, clean_data = clean_individual_type(
+                vintage, bldg_type, prog['space_type'], prog)
+            program_type_dict[new_name] = clean_data
+
+    # write the index data
+    program_type_index = os.path.join(dest_directory, '{}_index.json'.format(vintage))
```

Comment:
I mentioned what this file does above and I think the answer to your question is "yes" but we want to be clear about the terminology we use. I would instead say that this file is "an index of all room PorgramType names organized by building type". The first work that I use there (index) is where the file name comes from but we can use a synonym if you are worried about confusing it with the Python index. Maybe something like `collection` or `registry` instead of `index`.

Reply:
I kinda like `registry`. We haven't used that name anywhere else in honeybee so I think it has the lowest risk of confusion.

Reply:
`registry` or `collection` are both good.